WooCommerce/NotificationExtension/Info.plist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSExtension</key>
+	<dict>
+		<key>NSExtensionAttributes</key>
+		<dict>
+			<key>UNNotificationExtensionCategory</key>
+			<string>store_order</string>
```

Comment:
I'm curious about how only order notifications are routed to the notification extension, is this how notifications of type `store_order` come to this?

Reply:
All push notifications that we send come with a `type` key. In this case `"type": "store_order"`
By setting that category in the extension info plist in the `UNNotificationExtensionCategory` we guarantee that only those type of notifications are routed to the notification extension.

Does that make sense?

Reply:
Yup makes sense, thanks for the explanation!